.github/workflows/skywalking-eyes.yml

```diff
@@ -0,0 +1,33 @@
+name: Check License Header
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref_name }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  check-license-header:
+    name: Copyright Check
+    if: github.repository == 'uxlfoundation/oneDAL'
+    runs-on: ubuntu-24.04
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
Same question as in uxlfoundation/scikit-learn-intelex#2215

Reply:
The check can leave comments on the pull request if it finds issues. For this, write permissions need to be granted for these scopes.